MIP41/mip41.md

```diff
@@ -142,29 +142,54 @@ MIP41c5 subproposals must use the template located at [MIP41c5-Subproposal-Templ
 
 ---
 
-### MIP41c6: Interim Facilitators and the Fallback Decision Process
+### MIP41c6: Interim Facilitators
 
 #### Interim Facilitators
 
-Whenever a Core Unit finds itself with no Facilitator, an Interim Facilitator is needed to fill any gaps. The process to chose the Interim Facilitator is called "The Fallback Decision Process" and it is detailed in the heading below. However an interim facilitator is selected, Interim facilitators should clearly state their intended time of service. A two month minimum is recommended, with the option to renew.
+Whenever a Core Unit finds itself with no Facilitator, an Interim Facilitator is needed to ensure the continuity or graceful wind-down of the Core Unit.
 
-If a Retiring Facilitator is willing, they may nominate an Interim Facilitator. Subject to the nominated Interim Facilitator's acceptance, they are ratified as the new Facilitator by a simple majority vote of all active facilitators.
+A Retiring Faciliator may nominate a single Interim Facilitator. Subject to the nominated Interim Facilitator's acceptance, they are immediately ratified as the Interim Facilitator.
 
-In the event of a Core Unit offboarding, if the Retiring Facilitator is uncooperative, the Interim Facilitator will be in charge of overseeing/executing the unwinding process. In the event of a Facilitator offboarding, they will be in charge of ensuring the functional continuity of the Core Unit until a new permanent Facilitator is ratified.
+The Facilitators of other Core Units may elect an alternative Interim Facilitator within 72 hours of the initial Facilitator Offboarding using the Interim Facilitator Election Process. In the event an alternative Interim Facilitator is elected, they immediately replace the current Interim Facilitator, if one exists.
```

Comment:
Is this going to be the standard process if there is either no interim nominated by the retiring facilitator, or the previous facilitator has been offboarded by Governance without a new interim being proposed? If so, I'd state it explicitly

Reply:
This is covered in edge cases, I wanted to separate the 'but what if this happens' sections away from the general flow stuff.

Reply:
I should really read the whole thing before I start commenting